desc/objectives/_coils.py

```diff
@@ -1762,6 +1763,201 @@ def compute(self, params_1, params_2=None, constants=None):
         return Psi
 
 
+class LinkingCurrentConsistency(_Objective):
+    """Target the self-consistent poloidal linking current between the plasma and coils.
+
+    A self-consistent coil + plasma configuration must have the sum of the signed
+    currents in the coils that poloidally link the plasma equal to the total poloidal
+    current in the plasma G. This objective computes the difference between these two
```

Comment:
```suggestion
    current required to be linked by the plasma according to the loop integral of its toroidal magnetic field, given by `G`. This objective computes the difference between these two
```

`G` is technically not the current in the plasma but rather the poloidal linking current enclosed (i.e. thru the torus hole) by the plasma, as dictated by Ampere's law, and is a function just of the toroidal magnetic field of the plasma and the boundary geometry for the integral, not the plasma's poloidal current.